content/en/docs/tasks/traffic-management/egress/egress-tls-origination/snips.sh

```diff
@@ -118,11 +118,81 @@ HTTP/2 200
 ...
 ENDSNIP
 
-snip_cleanup_1() {
+snip_cleanup_the_tls_origination_configuration_1() {
 kubectl delete serviceentry edition-cnn-com
 kubectl delete destinationrule edition-cnn-com
 }
 
-snip_cleanup_2() {
-kubectl delete -f samples/sleep/sleep.yaml
+snip_configure_the_client_sleep_pod_1() {
+kubectl create secret generic client-credential --from-file=tls.key=client.example.com.key \
+  --from-file=tls.crt=client.example.com.crt --from-file=ca.crt=example.com.crt
+}
+
+snip_configure_the_client_sleep_pod_2() {
+kubectl create role client-credential-role --resource=secret --verb=get,list,watch
+kubectl create rolebinding client-credential-role-binding --role=client-credential-role --serviceaccount=default:sleep
+}
+
+snip_configure_mutual_tls_origination_for_egress_traffic_at_sidecar_1() {
+kubectl apply -f - <<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: originate-mtls-for-nginx
+spec:
+  workloadSelector:
+    matchLabels:
+      app: sleep
+  host: my-nginx.mesh-external.svc.cluster.local
+  trafficPolicy:
+    loadBalancer:
+      simple: ROUND_ROBIN
+    portLevelSettings:
+    - port:
+        number: 443
+      tls:
+        mode: MUTUAL
+        credentialName: client-credential # this must match the secret created earlier to hold client certs, and works only when DR has a workloadSelector
+        sni: my-nginx.mesh-external.svc.cluster.local
```

Comment:
this sni is optional. 
you can remove it or add a comment for it to avoid confusing.